CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.7] - 2023-02-21
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.6...v0.3.7)
+
+### Fixed
+* Contextual tuple propagation in the unoptimized ListObjects implementation (#565)
+
```

Comment:
Do we want to mention the nightly release fix?

Reply:
Let's not add anything related to nightlies until we confirm they are working. So far we don't have nightly builds.

I'll remove that from the `v0.3.6` announcement